### ISSUE TITLE
Breaking filled fluid pipes crashes the game

### ISSUE THREAD

**Issue by SuperFanatic** [refinedmods/refinedpipes#63]
Basically the title. Every single time I have to break a fluid pipe with fluid in it, the game freezes for a few seconds and then - desktop.

**Comment by chris5022** [comment]
same problem

**Comment by AgentK9** [comment]
I had the same problem. See error log here: https://gist.github.com/AgentK9/12240456ea3e076562b89f332bc36a0a.
Modpack: Stacia Expert Mode
Steps to recreate:
1. Build 5x5x1 TiC Smeltery in the nether, below a blaze spawner
2. Enable it by filling it with lava
3. Drop blazes and (unintentionally) wither skeletons into the smeltery
4. Use a basic fluid pipe and basic (insert name of the item that brings items/fluids out of an inventory here) on a smeltery drain to suck out the liquid blaze into an ender tank
5. Accidentally also suck out the milk
6. Break the pipe with a pickaxe
7. Corrupt your world (see screenshot: <img width="1680" alt="Screen Shot 2021-06-26 at 10 52 45 AM" src="https://user-images.githubusercontent.com/11563088/123517204-aef4f300-d66d-11eb-89f5-44a92050746d.png">)
8. ???
9. Profit!

**Comment by Jan-aka-q0** [comment]
When testing this out it seems to happen when interacting with filled fluid pipes

Here's my log
https://pastebin.com/raw/GShuDjEP

**Comment by mc-ganduron** [comment]
I'm not associated with the mod, but I wanted to mention that both logs posted here are erroring out in Ferrite Core's `FastMapStateHolderMixin` (end of the second line):

    java.lang.IllegalArgumentException: Cannot get property BooleanProperty{name=north, clazz=class java.lang.Boolean, values=[true, false]} as it does not exist in Block{minecraft:air}
	    at net.minecraft.state.StateHolder.func_177229_b(SourceFile:95) ~[?:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:ferritecore.fastmap.mixin.json:FastMapStateHolderMixin,pl:mixin:A}

* Is anyone having this problem *without* Ferrite Core installed?
* Could you post your Ferrite Core config file?

I'm not able to reproduce the problem (with or without Ferrite Core) so additional information would help.

**Comment by SuperFanatic** [comment]
I have never had ferrite core.

**Comment by mc-ganduron** [comment]
@SuperFanatic Can you reproduce the crash and post the resulting log file?

**Comment by mc-ganduron** [comment]
Also, this may have been fixed with https://github.com/refinedmods/refinedpipes/commit/da962f2420f26be9d98e0bb3c2bcb99f2062ddca for issue #12 (similar problem)... But the fix is part of version 0.5.1, which has not been released. ☹️ 

Hey @raoulvdberge, any word when 0.5.1 will be pushed to CurseForge?

**Comment by SheepReaper** [comment]
I've encountered this as well. liquid starlight in my case. Crashes my client and the clients of anyone within render range. Not totally sure that it's the mod's fault since other packs I've played have this mod and it functions correctly (pipe is picked up, and fluid spills out).

Stacia Expert modpack (I can't find their Github, but reported it on their discord)

crash dump: https://gist.githubusercontent.com/natanfudge/39f6820ca5fcedbbe5f9a7b7ce84f50e/raw/32110419a67d3d7bb7dcc90a5d4fd3e0ce4d7cb2/crash.txt
